### ISSUE TITLE
Do you know any hacky way to get this virtual stroll to work with elements ui tables filter-method?

### ISSUE THREAD

**Issue by willtham1** [xiaocheng555/el-table-virtual-scroll#16]

**Comment by xiaocheng555** [comment]
> _No description provided._

<img width="1739" alt="image" src="https://user-images.githubusercontent.com/19643785/232958438-a6f7446b-2b46-4b2c-b2d5-ba77321449f1.png">
I add a filter demo and you can see how the demo is written.

url：https://xiaocheng555.github.io/el-table-virtual-scroll/#/SortDemo?code=SortDemo&codeTitle=%E6%8E%92%E5%BA%8F%26%E8%BF%87%E6%BB%A4

**Comment by willtham1** [comment]
Wow you implemented that quick nice! I got the feature to work easily.
Added `@filter-change="onFilterChange"` to my el-table;
```
  methods: {
    onFilterChange(val) {
      if (val.statuss.length) {
        this.tableData = this.originalData.filter(item => {
          return val.statuss.some(status => item.status === status)
        })
      } else {
        this.tableData = this.originalData
      }
    },
},
  created() {
    this.originalData = [...this.tableData]
  },
```
This was used with the implementation of a vuex module containing my array. 
I created a copy of the array within the `created()` then passed `val` into the function which contains the filter option to the function(This is obtained by my `column-key='statuss'`). In addition for my way of re-populating the original data I used an `else` to reset the table.
This is for anyone who needs another example to use besides xiao's!!!
